packages/@uppy/dashboard/src/components/FileItem/FileProgress/index.js

```diff
@@ -51,7 +51,7 @@ module.exports = function FileProgress (props) {
   ) {
     return (
       <div class="uppy-DashboardItem-progress">
-        <div class="uppy-DashboardItem-progressIndicator">
+        <div class="uppy-u-reset uppy-DashboardItem-progressIndicator">
```

Comment:
Is `uppy-u-reset` supposed to be on a div, instead of a button?

Reply:
I think it's not needed, added it automatically after removing `@include .reset-button`, I'll remove it, thanks!